types/debug/index.d.ts

```diff
@@ -38,7 +38,7 @@ declare namespace debug {
         (formatter: any, ...args: any[]): void;
 
         enabled: boolean;
-        log: (v: any) => string;
+        log: (args: any[]) => any;
```

Comment:
I'm not sure about this change. This is actually a reference to `console.debug` or `console.log` in the library source.
Now if I'm looking at the official TypeScript lib.dom.d.ts, the definition of both functions are as followed:
```
log(message?: any, ...optionalParams: any[]): void;
```
But at the same time, with the debug library you should be able to define this as any function you like. Therefore I would either use the same definition from TS or at least change your change to have `...` before args, since requiring `args` to be an array the way you did is actually wrong in my opinion.

Reply:
Of course. you are right.
I've just fixed it.